commerce-account-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -26,7 +26,7 @@ List<CommerceAccount> commerceAccounts = commerceAccountDisplayContext.getCommer
 	<c:when test="<%= commerceAccountDisplayContext.getCommerceSiteType() == CommerceAccountConstants.SITE_TYPE_B2C %>">
 		<liferay-util:include page="/edit_user.jsp" servletContext="<%= application %>" />
 	</c:when>
-	<c:when test="<%= (commerceAccounts.size() > 1) || commerceAccountDisplayContext.hasManageCommerceAccountPermissions() %>">
+	<c:when test="<%= (commerceAccounts.size() > 1) && commerceAccountDisplayContext.hasManageCommerceAccountPermissions() %>">
 		<liferay-util:include page="/view_account_list.jsp" servletContext="<%= application %>" />
 	</c:when>
 	<c:when test="<%= !commerceAccounts.isEmpty() && commerceAccountDisplayContext.hasCommerceAccountModelPermissions(commerceAccounts.get(0), ActionKeys.VIEW) %>">
```

Comment:
With the change you've applied, if a user doesn't `hasManageCommerceAccountPermissions`, I think we should potentially check `hasCommerceAccountModelPermissions` for _each_ account in the list (if it's not empty). Before your change, we knew that if we got to the 3rd `when` test then there's only 1 account in the list, but we can't assume that to be true any more... @marco-leo what do you think?

Reply:
@ethanbustad  We do already this is CommerceAccountClayTable, that is the data provider for that list

Reply:
it just seems weird that we check `hasCommerceAccountModelPermissions` for the first element in the list before rendering `view_account.jsp` -- we don't know there's only one element in the list, so why single out the first element for a permission check?

Reply:
@alecsloan Thanks to this commerce, I just realized that the logic is (commerceAccounts.size() > 1) || commerceAccountDisplayContext.hasManageCommerceAccountPermissions() not &&

Reply:
So what do I need to do here? Change it back to or? I don't think that logic was correct.